apps/ai_tutor/storage/data/urls.txt

```diff
@@ -1,2 +1,2 @@
 https://tools4ds.github.io/fa2024/
-https://tools4ds.github.io/DS701-Course-Notes/
\ No newline at end of file
+https://raw.githubusercontent.com/tools4ds/ds701_fa2024_assignments/refs/heads/main/midterm/README.md
```

Comment:
Did you mean to remove the course notes? We still want to reference those as well.